### ISSUE TITLE
[Question] Can this be used to work directly between two raspberries

### ISSUE THREAD

**Issue by Esser50K** [emara-geek/secure-voice-chat#1]
Can this project be used to communicate between two raspberries instead of PC and raspberries?

I don't understand what the limitation would be if you used 2 raspberries?

I question this because the README explicitly says communication between a PC and a raspberry.

**Comment by tahaemara** [comment]
Yes, It could be run on two raspberries, I did not implement it with 2 raspberries because I have only one so, i tried it with pc and raspberry.
